TestResultSummaryService/DataManager.js

```diff
@@ -93,6 +93,13 @@ class DataManager {
             update.hasChildren = true;
             const result = await testResults.update( criteria, { $set: update } );
         } else if ( tests && tests.length > 0 ) {
+            let buildOutputId = null;
+            let response = await outputDB.populateDB( { output: output } );
+            if ( response && response.insertedCount === 1 ) {
+                buildOutputId = response.insertedIds[0];
+            }
+            update.buildOutputId = buildOutputId;

```

Comment:
If there is `tests` obj return, the test output will be stored in line 106. There is no need to store the output again. If the plan is to store the perf test output, then the perf parser should be updated to store the value as testOutput.

Reply:
Yes, I agree, changed it back already.